hack/python-sdk/gen-sdk.sh

```diff
@@ -44,8 +44,27 @@ fi
 
 if [ -z `which java` ]; then
   echo "Installing OpenJDK 11"
+
   apt update -y
-  apt-get install openjdk-11-jdk -y
+
+  apt-get install -y wget unzip zip git autoconf build-essential \
```

Comment:
The idea of installing from source was to decouple the installation process from environment specific package managers like `apt` - any way we can do this without using apt?

Reply:
@danielvegamyhre 
Thank you for your review. I will remove package-manager related part.